java/src/main/java/com/logicalclocks/hsfs/FeatureView.java

```diff
@@ -171,22 +173,41 @@ public String getBatchQuery(String startTime, String endTime)
     return featureViewEngine.getBatchQueryString(
```

Comment:
instead of calling `featureViewEngine.getBatchQueryString` could be better to call `this.getBatchQuery` to avoid duplication.